.browserslistrc

```diff
@@ -0,0 +1,2 @@
+defaults
```

Comment:
If we are doing a proper build process, I'd recommend we use the WP module for this that way we know we're getting the same as core. https://developer.wordpress.org/block-editor/reference-guides/packages/packages-browserslist-config/

Reply:
Good point! it's important that the settings are the same as for the core so that there will be no discrepancies when we take it out of webpack... will fix